libafl/src/monitors/tui/ui.rs

```diff
@@ -63,15 +63,19 @@ impl TuiUI {
 
     pub fn on_right(&mut self) {
         // never 0
-        self.clients_idx = 1 + self.clients_idx % (self.clients - 1);
+        if self.clients != 0 {
+            self.clients_idx = 1 + self.clients_idx % (self.clients - 1);
+        }
     }
 
     pub fn on_left(&mut self) {
         // never 0
-        if self.clients_idx == 1 {
-            self.clients_idx = self.clients - 1;
-        } else {
-            self.clients_idx = 1 + (self.clients_idx - 2) % (self.clients - 1);
+        if self.clients != 0 {
```

Comment:
This could be moved to the else block of the if below to simplify

Reply:
I am not really sure if I get what you mean here

Reply:
Ah nevermind, confused clients and client_idx